aerospike/metadata.csv

```diff
@@ -457,15 +457,180 @@ aerospike.namespace.udf_sub_udf_error,gauge,,,,Number of failed udf sub-transact
+aerospike.namespace.from_proxy_read_not_found,gauge,,,,Number of read transactions proxied from another node that resulted in not found.,0,aerospike,
+aerospike.namespace.from_proxy_read_success,gauge,,,,Number of successful read transactions proxied from another node.,0,aerospike,
+aerospike.namespace.from_proxy_read_timeout,gauge,,,,Number of timeouts for read transactions proxied from another node.,0,aerospike,
+aerospike.namespace.from_proxy_tsvc_error,gauge,,,,"Number of transactions proxied from another node that failed in the transaction service, before attempting to handle the transaction.",0,aerospike,
+aerospike.namespace.from_proxy_tsvc_timeout,gauge,,,,"Number of transactions proxied from another node that timed out while in the transaction service, before attempting to handle the transaction.",0,aerospike,
+aerospike.namespace.from_proxy_udf_complete,gauge,,,,Number of successful udf transactions proxied from another node.,0,aerospike,
+aerospike.namespace.from_proxy_udf_error,gauge,,,,Number of errors for udf transactions proxied from another node.,0,aerospike,
+aerospike.namespace.from_proxy_udf_filtered_out,gauge,,,,Number of udf transactions proxied from another node that did not happen because the record was filtered out via a predicate expression.,0,aerospike,
+aerospike.namespace.from_proxy_udf_timeout,gauge,,,,Number of timeouts for udf transactions proxied from another node.,0,aerospike,
+aerospike.namespace.from_proxy_write_error,gauge,,,,Number of errors for write transactions proxied from another node. This includes xdr_from_proxy_write_error.,0,aerospike,
+aerospike.namespace.from_proxy_write_filtered_out,gauge,,,,Number of write transactions proxied from another node that did not happen because the record was filtered out via a predicate expression.,0,aerospike,
+aerospike.namespace.from_proxy_write_success,gauge,,,,Number of successful write transactions proxied from another node. This includes xdr_from_proxy_write_success.,0,aerospike,
+aerospike.namespace.from_proxy_write_timeout,gauge,,,,Number of timeouts for write transactions proxied from another node. This includes xdr_from_proxy_write_timeout.,0,aerospike,
+aerospike.namespace.index_flash_used_bytes,gauge,,,,Total bytes in-use on the mount(s) for the primary index used by this namespace on this node. Applies only to Enterprise Version configured with index-type flash.,0,aerospike,
+aerospike.namespace.index_flash_used_pct,gauge,,,,Percentage of the mount(s) in-use for the primary index used by this namespace on this node. Applies only to Enterprise Version configured with index-type flash.,0,aerospike,
+aerospike.namespace.index_pmem_used_bytes,gauge,,,,Total bytes in-use on the mount(s) for the primary index used by this namespace on this node. Applies only to Enterprise Version configured with index-type pmem.,0,aerospike,
```

Comment:
It'd be nice to add types when it's easy, like bytes.

Reply:
Sure, I'll run through them again!

Reply:
They should be updated now!